radar/2024-12-01/grafana.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Grafana"
```

Comment:
I don't mind putting it back on but Grafana is such a given that I think we dropped it from our radar at some point. Can't quite remember. Has been with us for ages and is here to stay and in that sense does not add too much value to add it again I would say. But it also does not really hurt.

Reply:
its not in code at least